docs/Limitations.md

```diff
@@ -0,0 +1,19 @@
+# Limitations 
+
+## Unity SDK
+### Headless Mode
+If you enable Headless mode, you will not be able to collect visual 
+observations from your agents.
+
+### Rendering Speed and Synchronization
+Currently the speed of the game physics can only be increased to 100x 
+real-time. The Academy also moves in time with FixedUpdate() rather than 
+Update(), so game behavior tied to frame updates may be out of sync. 
```

Comment:
so game behavior implemented in Update() may be out of sync with the Agent decision making. See [Execution Order of Event Functions](https://docs.unity3d.com/Manual/ExecutionOrder.html) for more information.

Reply:
Should here be 
"so game behavior implemented in FixedUpdate() may be out of sync with the Agent decision making. See Execution Order of Event Functions for more information."
?

Reply:
Got it, I will use this sentence then.